contracts/HubPool.sol

```diff
@@ -779,6 +787,7 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
     // Note this method does a lot and wraps together the sending of tokens and updating the pooled token trackers. This
     // is done as a gas saving so we don't need to iterate over the l1Tokens multiple times.
     function _sendTokensToChainAndUpdatePooledTokenTrackers(
+        address spokePool,
```

Comment:
I was going to say that this is redundant butI actually think the way you have it now is cheaper as you only read the state once.

Reply:
exactly